tor/src/main/scala/org/bitcoins/tor/Socks5Connection.scala

```diff
@@ -213,6 +173,59 @@ object Socks5Connection {
 
   def portToByteString(port: Int): ByteString =
     ByteString((port & 0x0000ff00) >> 8, port & 0x000000ff)
+
+  def parseGreetings(data: ByteString, passwordAuth: Boolean): Byte = {
```

Comment:
Do you have any references to the SOCKS5 specification you could link a scaladoc for future users of the code? I'm not sure where these magic bytes are coming from.